src/en/anchira/build.gradle

```diff
@@ -0,0 +1,17 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlinx-serialization'
+
+ext {
+    extName = 'Anchira'
+    pkgNameSuffix = 'en.anchira'
+    extClass = '.Anchira'
+    extVersionCode = 1
+    isNsfw = true
+}
+
+dependencies {
+    implementation "com.ensarsarajcic.kotlinx:serialization-msgpack:0.5.5"
```

Comment:
This seems unused. Is there anything msgpack-encoded?

Reply:
Another leftover. The API originally was encoded with MessagePack.